MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -2151,6 +2164,9 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 	} catch (cereal::Exception& ex) {
 		setConfigurationResult(reportError(VK_SUCCESS, "Error reading pipeline cache data: %s", ex.what()));
 	}
+#else
+	MVKAssert(false, "Pipeline cache serialization is unavailable. To enable pipeline cache serialization, build MoltenVK without the MVK_EXCLUDE_CEREAL build setting.");
```

Comment:
Similarly, I don't think `MVKAssert` is appropriate here. Probably better to return an error result:

	setConfigurationResult(reportError(VK_ERROR_INITIALIZATION_FAILED, ...

While you're at it...can you change the the existing return of `VK_SUCCESS` to `VK_ERROR_INITIALIZATION_FAILED`, as well, please, as that existing return value looks to be a mistake.

Reply:
Ok, done.